specification/azsadmin/resource-manager/user-subscriptions/Microsoft.Subscriptions/preview/2015-11-01/Subscriptions.json

```diff
@@ -0,0 +1,412 @@
+                    }
+              },
+              "tags": [
+                "Subscriptions"
+              ],
+              "operationId": "Subscriptions_Get",
+              "description": "Gets details about particular subscription.",
+              "parameters": [
+                {
+                  "name": "subscriptionId",
+                  "in": "path",
+                  "required": true,
+                  "type": "string",
+                  "description": "Id of the subscription."
+                },
+                {
+                  "$ref": "#/parameters/ApiVersionParameter"
+                }
+              ],
+              "responses": {
+                "200": {
+                  "description": "",
+                  "schema": {
+                    "$ref": "#/definitions/Subscription"
+                  }
+                }
+              }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create or updates a subscription": {
+                        "$ref": "./examples/Subscriptions/Create.json"
+                    }
+                },
+                "description": "Create or updates a subscription.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "Subscriptions_CreateOrUpdate",
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "Id of the subscription."
+                    },
+                    {
+                        "$ref": "#/parameters/NewSubscriptionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Created",
+                        "schema": {
+                            "$ref": "#/definitions/Subscription"
+                        }
+                    },
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Subscription"
+                        }
+                      }
+                    }
+                },
+                "delete": {
+                    "x-ms-examples": {
+                        "Delete the specifed subscription": {
+                            "$ref": "./examples/Subscriptions/Delete.json"
+                        }
+                    },
+                    "description": "Delete the specifed subscription.",
+                    "tags": [
+                        "Subscriptions"
+                    ],
+                    "operationId": "Subscriptions_Delete",
+                    "parameters": [
```

Comment:
Why are you not using the SubscriptionIdParameter here?

Reply:
Subscription is the main resource for this spec, I dont want to mask it in global param